### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by rimvydasrub**
Adding structure for environment setup and task generation

### CODE REVIEW

File: rearrangement_benchmark/tasks/move_task.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from .rearrangement_task import RearrangementTask
```
Comment (peterdavidfagan) [inline rearrangement_benchmark/tasks/move_task.py:1]: I think we should generally avoid relative imports.
```suggestion
from rearrangement_benchmark.rearrangement_task import RearrangementTask
```

File: rearrangement_benchmark/tasks/stack_task.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from .rearrangement_task import RearrangementTask
```
Comment (peterdavidfagan) [inline rearrangement_benchmark/tasks/stack_task.py:1]: I think we should generally avoid relative imports.
```suggestion
from rearrangement_benchmark.rearrangement_task import RearrangementTask
```

**Review by peterdavidfagan — COMMENTED** [review]
I agree with structural changes, request to amend relative imports made.

**Comment by peterdavidfagan** [comment]
The following import requires fixing:

https://github.com/peterdavidfagan/rearrangement_benchmark/blob/3f58fbf9951cfbe3d967ddb0bfacf0ff58e04d25/rearrangement_benchmark/transporter_network_data_generation.py#L11

**Comment by rimvydasrub** [comment]
@peterdavidfagan see the suggested updates for the import fixes

**Comment by peterdavidfagan** [comment]
@rimvydasrub, why are move and stack tasks split into different scripts? I assume we have one API interface with pick/place action space etc. Clarification on reasoning for this is appreciated.

**Comment by rimvydasrub** [comment]
The initial idea was to have different scripts for different task types as they have different goal descriptions (move vs stack) and can have different logic in handling this. For instance, for a stack action, you should check that the objects manipulated are stackable (e.g. not stack a cube on a sphere).

We could have one API for both of the tasks, but I think it is better to separate the logic, especially if you want to have more tasks. e.g. push, scoop, etc.

**Comment by peterdavidfagan** [comment]
Great this makes sense and sounds like good design. This aligns quite well with MoMa so should work well (once I get operating over abstract action spaces working).

**Pull request merged**